src/pk-transaction.c

```diff
@@ -94,7 +97,8 @@ struct PkTransactionPrivate
 	gboolean		 emit_media_change_required;
 	gboolean		 caller_active;
 	gboolean		 exclusive;
-	guint			 uid;
+	guint32			 uid;
```

Comment:
Would it make sense to rename this to `client_uid` while you’re here?

Reply:
Probably... Just UID is actually a bit confusing for people new to the code, but I probably just got used to it.